pom.xml

```diff
@@ -43,7 +43,7 @@
         <lucene.version>5.5.2</lucene.version>
         <hamcrest.version>1.3</hamcrest.version>
         <mockito.version>2.28.2</mockito.version>
-        <jest.version>6.3.1</jest.version>
+        <jest.version>6.3.2-SNAPSHOT</jest.version>
```

Comment:
Testing used snapshot built from https://github.com/searchbox-io/Jest/pull/679 , so that `?pipeline=...` could be used as a test, and this is also the desired functionality.